app/views/PageParents.vue

```diff
@@ -66,7 +522,29 @@
 
       ...mapActions({
         loadProducts: 'products/loadProducts'
-      })
+      }),
+
+      openPremiumSubscribeModal () {
+        this.subscribeModalOpen = true
+      },
+
+      subscribeBasic () {
+        this.$refs.subscribeModal.$once('shown', () => {
+          const modal = this.$refs.subscribeModal.$data.modalViewInstance
+          modal.onClickPurchaseButton()
+        })
+
+        this.openPremiumSubscribeModal()
```

Comment:
Is this where both modals open. Is it intended for the modal to redirect to signing up if the user isn't logged in?
![8aed805d9d9ba6b8ea464ab77f83e657](https://user-images.githubusercontent.com/15080861/84720020-bf8a6d80-af31-11ea-9256-735998796d3d.gif)

Reply:
Yup, this is a known issue to ship this quickly.  We're basically opening the subscription modal and clicking the next step for the user.  Tried to avoid the first modal showing but ran into some race conditions if we did it any faster than thsi.